packages/database/src/migrations/20231120003955-CreateDuplicateSurveys-modifies-data.js

```diff
@@ -0,0 +1,234 @@
+/* eslint-disable camelcase */
+
+'use strict';
+
+import surveys from './migrationData/20231120003955-CreateDuplicateSurveys/surveys.json';
+import { generateId, insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+// get id of project with code newProjectCode
+const getProjectId = async (db, projectCode) => {
+  const projectId = await db.runSql(`SELECT id FROM project WHERE code = '${projectCode}'`);
+  return projectId.rows[0].id;
+};
+
+// get new survey code by appending project initials to old survey code
+const getNewSurveyCode = (oldSurveyCode, newProjectCode) => {
+  const projectInitials = newProjectCode
+    .split('_')
+    .map(word => word[0])
+    .join('');
+  return `${oldSurveyCode}_${projectInitials.toUpperCase()}`;
```

Comment:
This'll work fine, but also it's calculating from a list of data we know. So `newProjectCode` is always `dl_tamanu`, and the result is always `${oldSurveyCode}_DT` right?

Probably not worth editing, just pointing it out 😅